src/fastman/commands/project.py

```
@@ -0,0 +1,506 @@
+
+        elif package_manager == "poetry":
+            # Check if poetry is available
+            if not shutil.which("poetry"):
+                Output.warn("poetry not found. Please install poetry first.")
+                self._create_requirements_txt(dependencies)
+                return
+
+            Output.info("Initializing with poetry...")
+            try:
+                # Configure poetry to create venv in project
+                subprocess.run(["poetry", "config", "virtualenvs.in-project", "true"],
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)